.github/PENDING_REVIEW_NOTIFICATION_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Hi {{ username }},
+
+It looks like you haven't reviewed the following Oppia-web PRs within the expected time:
```

Comment:
Oppia-web is added here as we are going to have the same team for Ooppia-android and this is going to help us identify the project.